tests/v2/test_waku_rln_relay.nim

```diff
@@ -666,4 +723,29 @@ suite "Waku rln relay":
     doAssert(badVerifyIsSuccessful)
     # badF=1 means the proof is not verified
     # verification of the bad proof should fail
-    doAssert(badF == 1)
\ No newline at end of file
+    doAssert(badF == 1)
+  
+  test "create a list of membership keys and construct a Merkle tree based on the list":
+    let (list, root) = createMembershipList(100) 
+    debug "created membership key list", list
+    check: 
+      list.len == 200 # two parts for each membership key i.e., one id key and one id commitment
```

Comment:
If there are really 100 logical elements in this list, perhaps it should be stored as a tuple instead? So `[(key0, commitment0), (key1, commitment2), ...]`. Right now it isn't obvious to me how the 200 items are mapped

Reply:
Revised it to a list of 100 tuples, the new version should be more clear. Please let me know if there is unclarity.